src/fiatExchanges/FiatExchangeAmount.test.tsx

```diff
@@ -185,81 +175,10 @@ describe('FiatExchangeAmount cashIn', () => {
     expect(tree.getByTestId('FiatExchangeNextButton')).not.toBeDisabled()
   })
 
-  it('opens a dialog when the cUSD amount is lower than the limit', () => {
```

Comment:
We no longer have a 'minimum amount' since exchanges can handle very low amounts.